core/scripts/sql/texera_ddl.sql

```diff
@@ -112,4 +115,34 @@ CREATE TABLE IF NOT EXISTS workflow_version
     `creation_time`      TIMESTAMP                   NOT NULL DEFAULT CURRENT_TIMESTAMP,
     PRIMARY KEY (`vid`),
     FOREIGN KEY (`wid`) REFERENCES `workflow` (`wid`) ON DELETE CASCADE
+) ENGINE = INNODB;
+
+CREATE TABLE IF NOT EXISTS project
```

Comment:
I suggest name it to be `user_project` and globally we should use `UserProject` to reference it. `project` is just too general and would make conflicts in the future.

Reply:
done